### ISSUE TITLE
Visualize Feature Embeddings for 3D Models

### ISSUE THREAD

**Issue by albertotono** [tensorflow/tensorboard#4365]
It would be great to be able to visualize 3d models in the latent space with Tensorboard, navigating the space instead of having picture see the 3d models.

Something like this https://twitter.com/marian42_/status/1188969971898048512?lang=en

**Comment by stephanwlee** [comment]
Thanks for the feedback. Have you seen our embedding projector? (Demo at https://projector.tensorflow.org/). It surely is not as cool looking as the gif in the tweet but I believe it is what you are looking for. 

ICYMI, here is the tutorial for the projector plugin: https://www.tensorflow.org/tensorboard/tensorboard_projector_plugin

**Comment by albertotono** [comment]
Hi @stephanwlee , yes I was looking at that, but I haven't seen a way to display 3D shapes. Is it possible? 

Or as workaround, I can take a picture of the shape and use that to display in the t-SNE?

**Comment by stephanwlee** [comment]
I believe it is possible to show 2d image!

If you use the left nav and select "Mnist with Image", you will see an example it showing MNIST image.

![image](https://user-images.githubusercontent.com/2547313/99766497-120a2000-2ab6-11eb-8b07-af05de925d1d.png)

**Comment by albertotono** [comment]
Yes, I know the possibility to display images, I am currently using that as workaround, but it would be nice to have 3d models.

**Comment by stephanwlee** [comment]
> Or as workaround, I can take a picture of the shape and use that to display in the t-SNE.

Got it, I was not sure whether you were acknowledging the possibility in current tool. 

While I captured this as a feature request, I cannot imagine this being compelling for large number of our users and make it to our priority list.
